dist/s_style

```diff
@@ -57,10 +57,19 @@ else
 
 	cmp $t $f > /dev/null 2>&1 || (echo "modifying $f" && cp $t $f)
 
-	egrep -w 'a a|an an|and and|are are|be be|by by|for for|from from|if if|in in[[:space:]]|is is|it it|of of|the the|this this|to to|was was|were were|when when|with with|a an|an a|a the|the a' $f > $t
+	# Finding paired typos in the comments of different file types and excluding invalid edge cases.
+	if [ "${fname##*.}" = "py" ]; then
+		egrep '#.*\b([a-zA-Z]+)\s\1\b' $f | egrep -v -e "in-memory" -e "one-file" > $t
```

Comment:
np: Do we need the `\b` here? My regex isn't too good, so want to understand more :)

Reply:
The `\b` represents a word boundary that only matches a non-word character going to a word character. This ensures that matches are a whole word by themselves and not part of another word. It's useful to avoid instances like matching on "connection on" where the "on" from both words would match!

Reply:
The \b represents a word boundary that only matches a non-word character going to a word character. This ensures that matches are a whole word by themselves and not part of another word. It's useful to avoid instances like matching on "connection on" where the "on" from both words would match!